### PR TITLE
LN period neuron updates

### LINKED ISSUE #164: New subtype of l-LNv neuron and expanded descriptions of lateral neurons
FBrf0238313 defines a new subtype of l-LNv neuron  the authors call 'l-LNv extra'. The convention seems to be that this should be called 'extra I-LNv neuron' based on 'extra LNd neuron'.

The authors also expand on the available morphological descriptions for most lateral neurons.

### PR DESCRIPTION
**Pull request by Clare72**
'Extra l-LNv neuron', more LNd neuron subclasses, more relationships for 'Pdf negative s-LNv neuron'

Fixes #164

### CODE REVIEW

File: fbbt/src/ontology/fbbt-edit.obo [hunk 1]
```diff
@@ -68431,6 +68440,82 @@ synonym: "mVAC" EXACT []
...
+comment: Possible equivalence with 'extra LNd neuron', as both have a large cell body compared to other LNds (FlyBase:FBrf0238313 and FlyBase:FBrf0191052).
+synonym: "ITP and CRY positive LNd neuron" EXACT []
+synonym: "ITP expressing LNd neuron" EXACT []
+synonym: "ITP positive LNd neuron" EXACT []
+synonym: "ITP+ CRY+ LNd neuron" EXACT []
+synonym: "ITP+ LNd neuron" EXACT []
+synonym: "ITP+/CRY+ LNd neuron" EXACT []
+synonym: "ITP/CRY coexpressing LNd neuron" EXACT []
+is_a: FBbt:00004045 ! LNd neuron
+relationship: connected_to FBbt:00040042 ! posterior ventrolateral protocerebrum
```
Comment (dosumis) [inline fbbt/src/ontology/fbbt-edit.obo]: Why connected_to ?  This relationship is quite abstract and has not been used very much up to now.  Not sure it makes sense for a neuron.

File: fbbt/src/ontology/fbbt-edit.obo [hunk 2]
```diff
@@ -76559,7 +76644,7 @@ creation_date: 2010-07-14T02:46:14Z
 id: FBbt:00100170
 name: extra LNd neuron
 def: "LNd neuron of the adult brain characterized by having a slightly larger nucleus and cytoplasmic area compared to the other LNd neurons." [FlyBase:FBrf0191052]
-comment: Rieger et al., (2006) comment that the extra LNd also stains more strongly for Period (FBgn0003068) and Timeless (FBgn0014396) compared to the other LNd neurons. Under constant light (LL) conditions, the extra LNd neuron is the only LNd neuron which shows significant Period and Timeless cycling.
+comment: Rieger et al., (2006) comment that the extra LNd also stains more strongly for Period (FBgn0003068) and Timeless (FBgn0014396) compared to the other LNd neurons. Under constant light (LL) conditions, the extra LNd neuron is the only LNd neuron which shows significant Period and Timeless cycling.\n\nThis neuron must belong to one of the other LNd subclasses. It is most likely equivalent to the ITP and CRY expressing LNd neuron, as this has the largest cell diameter among LNd neurons (Schubert et al., 2017 - FlyBase:FBrf0238313).
```
Comment (dosumis) [inline fbbt/src/ontology/fbbt-edit.obo]: Minor formatting issue: please avoid adding returns in text fields.  (newline appears as "\n" above.)

**Comment by dosumis** [comment]
Looks great! Very detailed.  I just have a couple of small quibbles - added in-line on [diffs](https://github.com/FlyBase/drosophila-anatomy-developmental-ontology/pull/205/files)

**Comment by Clare72** [comment]
There seems to be something wrong with the imports - I can see them in the file in Aquamacs, but they do not load in Protege. If I try to add them back manually in protege, it can't save:
>Reason: Could not find an ontology storer which can handle the format: OBO 1.2 Format

**Comment by dosumis** [comment]
Not sure what's going on. I see the same Phenotype in Protege.  Looks like travis is failing to load the ontology file: [File fbbt-edit.obo is not a valid ontology](https://travis-ci.org/FlyBase/drosophila-anatomy-developmental-ontology/builds/382739801?utm_source=github_status&utm_medium=notification).

Investigating now.

**Comment by dosumis** [comment]
This seems to have done the trick: 08d8280eabb58b37d67c75a66ce6b2558f145308.  Now loading in Protege and Travis.

**Comment by Clare72** [comment]
Great, thanks! Must have mis-typed during a merge.

File: fbbt/src/ontology/fbbt-edit.obo (hunk 1 above)
Comment (dosumis) [reply fbbt/src/ontology/fbbt-edit.obo]: Good point. I still think best to remove.  I'm not sure the meaning will be clear to users.  Also - this interpretation doesn't quite fit with the intention of the relation.  A 'fiat boundary' is a one we declare by *fiat* that has no physical correlate. The PVLP boundary might be considered fiat, but the boundary of the neuron projection is its cell membrane which is not.  One use case for this relationship between tracts/nerves and neuropils.  Here there is typically a fiat boundary marking the end of the tract and the beginning of the neuropil that cuts through the neuron projections in the tract.  Another use case is where a tract splits and each branch gets a new name.  I think these two uses are reasonably intuitive to users.

**Comment by dosumis** [comment]
Looks good.  Please resolve conflicts & merge. Ta.

**Pull request merged**